cmd/rpcdaemon/filters/filters.go

```diff
@@ -224,7 +224,7 @@ func (ff *Filters) subscribeToPendingBlocks(ctx context.Context, mining txpool.M
 
 func (ff *Filters) HandlePendingBlock(reply *txpool.OnPendingBlockReply) {
 	b := &types.Block{}
-	if len(reply.RplBlock) == 0 {
+	if reply == nil || len(reply.RplBlock) == 0 {
```

Comment:
why not just `reply == nil`?

Reply:
`len(reply.RplBlock) == 0` was the previous check, I left it as is. But on my Erigon instance `reply` was `nil` during shutdown, thus this fix.